<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTitleBar</class>
 <widget class="QWidget" name="CTitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CTitleBar</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_logo">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_title">
     <property name="styleSheet">
      <string notr="true">QLabel
{
	font-family:&quot;Microsoft YaHei&quot;;
	font-size:18px;
	color:rgb(240,230,220);
}</string>
     </property>
     <property name="text">
      <string>录制源</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnSkin">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-image:url(:/DesktopRec/picture/skin.svg)
}

QPushButton:hover
{
	border-image:url(:/DesktopRec/picture/skin_hover.svg)
}

</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSet">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-image:url(:/DesktopRec/picture/set.svg)
}

QPushButton:hover
{
	border-image:url(:/DesktopRec/picture/set_hover.svg)
}

</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnMin">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-image:url(:/DesktopRec/picture/min.svg)
}

QPushButton:hover
{
	border-image:url(:/DesktopRec/picture/min_hover.svg)
}

</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnClose">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-image:url(:/DesktopRec/picture/close.svg)
}

QPushButton:hover
{
	border-image:url(:/DesktopRec/picture/close_hover.svg)
}

</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
